app/src/main/java/com/android/unio/ui/event/EventDetails.kt

```diff
@@ -158,7 +160,9 @@ fun EventScreenScaffold(
                         imageVector =
                             if (isSaved) Icons.Rounded.Favorite else Icons.Rounded.FavoriteBorder,
                         contentDescription = if (isSaved) "Saved" else "Not saved",
-                        tint = if (isSaved) Color.Red else Color.White)
+                        tint =
+                            if (isSaved) MaterialTheme.colorScheme.error
```

Comment:
True, that's a fair statement. Also, it's actually not exactly the same color and although I prefer the `Color.Red`, I decided to change it to this for compliance with the Material Theme. I might revert this change, what do you think ?

Reply:
I think that in this case it would be fine to keep `Color.red`, but otherwise we could also maybe add something along the lines of `MaterialTheme.colorScheme.like`